exampleCourse/questions/demo/annotated/mathNumbersConcept-all/info.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
The QID here is using a mixture of camelCase and snake-case (and similarly for other `mathNumbersConcept` questions).